docs/user/index.rst

```diff
@@ -10,6 +10,7 @@ Read the Docs: documentation simplified
    /intro/getting-started-with-sphinx
    /intro/getting-started-with-mkdocs
    /intro/import-guide
+   /config-file/index
```

Comment:
I think we should add this link into the index content, as well as the sidebar?

Reply:
Ah yes, good call. I found it easy to miss all of these places where we've duplicated navigation, like the index pages. It's too bad `toctree` isn't more extendable or flexible somehow, it could be nice to author this content just using `toctree`.

Reply:
It looks like all tutorials are listed on `/tutorials/index` instead of on `/index`. This link was also missing there, so I added it there instead of on `/index`.

I'm not quite sure where I am on the `/index` page. I'll stick with the pattern we have now, but fully agree it's not ideal. Those tutorials should be top level content, they get  most of our traffic.

Reply:
We have a listing page for all the pieces of content in each section, which is the tutorials/index. That is definitely where we should be auto-generating the listing, but we now have all the content duplicated like 3 different places 🙃

Reply:
Yeah that `/tutorials/index` would be a good one to automate, and the `/index` could either reuse that, or could be curated if we want -- though I suspect we'll want all the tutorials surfaced pretty high up, they are important content.

I'll get this merged for now, but definitely feel free to muck around further on the navigation. Sounds like we're looking for the same thing from `/index` and friends.